CMakeLists.txt

```
@@ -13,6 +13,8 @@ project(TNT)
 # ==================================================================================================
 option(FILAMENT_USE_EXTERNAL_GLES3 "Experimental: Compile Filament against OpenGL ES 3" OFF)
 
+option(FILAMENT_USE_ANGLE "Compile Filament against ANGLE" OFF)
```

```suggestion
option(FILAMENT_USE_ANGLE "Compile Filament against ANGLE (Windows only!)" OFF)
```
In theory one could use ANGLE on Linux, macOS too. But the way we integrate ANGLE into Filament assumes that we use it only on Windows